server/graphql/schema.graphql

```diff
@@ -0,0 +1,129 @@
+type BookReview {
+  id: ID!
+  review: String
+  rating: Int!
+  reviewer_email: String!
+  created_at: String!
+  book: Book!
+}
+
+type Book {
```

Comment:
why not use Node interface on any schema?
eg:- `type Book implements Node`

Reply:
used Node but didn't find why it is used anywhere a solid explanation on internet, can you please share some resource?

Reply:
You might want to check caching in apollo this will introduce node